docs/io.cozy.identities.md

```diff
@@ -0,0 +1,57 @@
+[Table of contents](README.md#table-of-contents)
+
+# Identities doctype
+
+## `io.cozy.identities`
+
+### Description
+- Represents the differents identities fetched by datasources (connectors most of the time)
```

Comment:
I'm not sure I understand the purpose of this doctype. Is it for the several identities of the cozy owner? Or are they for other people (and, in that case, why not use the io.cozy.contacts)?

Reply:
It's contact addresses and data from your bills and connectors accounts. Most are about you, but it could be about someone else, and it could be obsolete information.

Put everything in io.cozy.contacts mean that you will have dozen duplicates, that you won't be able to sort. Furthermore, if today we only retrieve contact information, we aim at more data.

The purpose is to have this intermediary "identities" fetched from everywhere, and a code to fill a unique "me" document in io.cozy.contacts